Initial commit of script for wat_067_rw0_aqueduct_riverine_flood_hazard (#189)
resource-watch/data-pre-processing: req_017_thailand_flooding/README.md

## Thailand Flooding Pre-processing

> This file describes the data pre-processing that was done to create the [Thailand Flooding dataset](https://resourcewatch.org/admin/data/datasets/0fb4086b-37d7-464d-9eaf-b9104d8992ad) used as a swipey for the [Climate-related Physical Risks](https://resourcewatch.org/dashboards/408).

Reviewer: ```suggestion
This file describes the data pre-processing that was done to create the [Thailand Flooding dataset]() for display on [Resource Watch](https://resourcewatch.org/embed/map-swipe?zoom=10.124765919920378&lat=14.348293168896923&lng=100.57237782742156&layers=065b68fa-19af-4381-b5bf-8dd31d0a1309,23f38c8a-d7dc-4549-9e72-da5c576f4c31) to be included in the [Climate-related Physical Risks dashboard](https://resourcewatch.org/dashboards/408).
```

> This file describes the data pre-processing that was done to create the [Thailand Flooding dataset](https://resourcewatch.org/admin/data/datasets/0fb4086b-37d7-464d-9eaf-b9104d8992ad) used as a swipey for the [Climate-related Physical Risks](https://resourcewatch.org/dashboards/408).

Reviewer: has dashboard url changed?

> This file describes the data pre-processing that was done to create the [Thailand Flooding dataset](https://resourcewatch.org/admin/data/datasets/0fb4086b-37d7-464d-9eaf-b9104d8992ad) used as a swipey for the [Climate-related Physical Risks](https://resourcewatch.org/dashboards/408).

Author: ```suggestion
This file describes the data pre-processing that was done to create the [Thailand Flooding dataset](https://earthobservatory.nasa.gov/images/76234/floods-swamp-historic-city-in-thailand) for display on [Resource Watch](https://resourcewatch.org/embed/map-swipe?zoom=10.124765919920378&lat=14.348293168896923&lng=100.57237782742156&layers=065b68fa-19af-4381-b5bf-8dd31d0a1309,23f38c8a-d7dc-4549-9e72-da5c576f4c31) to be included in the [Climate-related Physical Risks dashboard](https://resourcewatch.org/dashboards/climate-related-physical-risks).
```

> The modified files shown on the dashboard were provided by [Lihuan Zhou](https://www.wri.org/profile/lihuan-zhou) from the Finance Center via email. They were then uploaded to Google Earth Engine. The original files were modified by Lihuan to include two white rectangles and text, highlighting nearby Sony and Honda factories that were affected by the floods.
> 
> The original files are from the Advanced Land Imager (ALI) on NASA's Earth Observing-1 satellite and the United States geological Survey. These images are available for display and download on NASA's Earth Observatory [Image of the Day](https://earthobservatory.nasa.gov/images/76234/floods-swamp-historic-city-in-thailand) for October 27, 2011.

Reviewer: ```suggestion
This dataset contains satellite imagery, provided by the source as geotiff files. These files come from the Advanced Land Imager (ALI) on NASA's Earth Observing-1 satellite and the United States Geological Survey. These images were modified to include two white rectangles and text, highlighting the Sony and Honda factories that were affected by the October 2011 floods in Thailand. These modifications were done by [Lihuan Zhou](https://www.wri.org/profile/lihuan-zhou) and shared with the Resource Watch team via e-mail. The modified images were then uploaded to Google Earth Engine so that they could be displayed on Resource Watch. The original images are available for display and download on NASA's Earth Observatory [Image of the Day](https://earthobservatory.nasa.gov/images/76234/floods-swamp-historic-city-in-thailand) for October 27, 2011.
```
